service/src/main/kotlin/fi/espoo/evaka/pis/service/FamilyOverviewService.kt

```diff
@@ -24,13 +25,14 @@ class FamilyOverviewService(private val objectMapper: ObjectMapper) {
             """
 WITH adult_ids AS
 (
-    SELECT :id id
+    SELECT :id AS id
+    
     UNION
-    SELECT DISTINCT fp2.person_id id 
+    
+    SELECT fp2.person_id AS id 
     FROM fridge_partner fp1
-    JOIN fridge_partner fp2 ON fp1.partnership_id = fp2.partnership_id
-    WHERE fp1.person_id = :id
-    AND now() BETWEEN fp1.start_date AND fp1.end_date
+    JOIN fridge_partner fp2 ON fp1.partnership_id = fp2.partnership_id AND fp1.person_id != fp2.person_id
```

Comment:
These can probably be conflicted, right?

Reply:
oh yeah, I thought about that too but then forgot :D